template/package.json

```diff
@@ -85,6 +90,13 @@
     {{/e2e}}
     "semver": "^5.3.0",
     "shelljs": "^0.7.6",
+    {{#stylelint}}
+    "stylelint": "github:stylelint/stylelint",
```

Comment:
I'm not sure there is a necessary to use github? it has no version control..I think.

Reply:
I'm waiting for the next release of stylelint that including stylelint/stylelint#2799.

Reply:
https://github.com/stylelint/stylelint/issues/2838
It's going to be released in couple of days.